aws-auth-cognito/src/main/java/com/amplifyframework/auth/cognito/helpers/FlutterFactory.kt

```diff
@@ -0,0 +1,34 @@
+package com.amplifyframework.auth.cognito.helpers
+
+import com.amplifyframework.auth.AWSCredentials
+import com.amplifyframework.auth.cognito.AWSCognitoAuthSession
+import com.amplifyframework.auth.cognito.AWSCognitoUserPoolTokens
+import com.amplifyframework.auth.result.AuthSessionResult
+
+/**
+ * Helper class that allows Amplify Flutter dev team to create necessary return types with internal constructors.
+ * This class should not be consumed outside of the Amplify Flutter development team and should not be considered
```

Comment:
```suggestion
 * Helper class that allows Amplify Flutter to create necessary return types with internal constructors.
 * This class should not be consumed outside of the Amplify Flutter and should not be considered
```

Reply:
Resolving with additional removal of extra "the"